src/main/resources/development.logger.xml

```diff
@@ -10,8 +10,8 @@
   <logger name="slick"                                                     level="INFO"  />
   <logger name="slick.ast"                                                 level="INFO"  />
   <logger name="slick.compiler"                                            level="INFO"  />
-  <logger name="slick.jdbc.JdbcBackend.statement"                          level="INFO" />
-  <logger name="slick.jdbc.StatementInvoker.result"                        level="INFO"  />
+  <logger name="slick.jdbc.JdbcBackend.statement"                          level="DEBUG" />
+  <logger name="slick.jdbc.StatementInvoker.result"                        level="DEBUG"  />
```

Comment:
yeah, forgot to set those back. Just pushed a change that puts them back to INFO